Makefile

```diff
@@ -36,6 +36,11 @@ unit:
 functional-s3:
 	pipenv run pytest tests/functional/s3
 
+functional-gcs:
+	# You will need to have the authentication set up to run these test
+	# 🚨🚨🚨WARNING🚨🚨🚨 -  This will create a bucket and file as configured in tests/functional/gcs/conftest.py
```

Comment:
Does this not also apply for the `s3` tests? If so, then don't add these comments, as there aren't comments like these for the `s3` ones.

Reply:
I am also not sure about this and depends on what the maintainers want in there tests.